remix/app/routes/_app._index.tsx

```diff
@@ -1,12 +1,19 @@
 import { User } from '@prisma/client';
 import { useOutletContext } from '@remix-run/react';
 
+import AppLink from '../components/AppLink';
+
 export default function Index() {
   const [user] = useOutletContext() as [User];
 
   return (
     <>
```

Comment:
I'm just curious but do we need the fragment here? Cause I thought fragments are used to just group childs that not have a parent, but we have a `section` parent here 🤔